### ISSUE TITLE
NPM Package Not Found

### ISSUE THREAD

**Issue by pyrophire** [anagram4wander/ng-vfor-lib#1]
**Describe the bug**
The documentation/Readme has "npm install ng-vfor-lib" but NPM returns a notfound for ng-vfor-lib. When installing "ngvforlib" it comes down as a tgz file with a package json.

**To Reproduce**
1. run "npm install ng-vfor-lib"
2. See error

**Expected behavior**
Package should return a series of folders/files containing library.

**Desktop (please complete the following information):**
 - OS: MacOS High Sierra

**Comment by mahajnamahmod** [comment]
same here, it appears to be under the name 'ngvforlib' rather than, 'ng-vfor-lib' problem is after installing the library i get the following  ERROR in Couldn't resolve original symbol for ./lib/core/directives/ng-vFor-container.directive from /Users/XXXXXXX/Desktop/starter-kit/node_modules/ngvforlib/package/ng-vfor-lib.d.ts

**Comment by OrangeDrangon** [comment]
I cannot even resolve the module after install

**Comment by anagram4wander** [comment]
> same here, it appears to be under the name 'ngvforlib' rather than, 'ng-vfor-lib' problem is after installing the library i get the following ERROR in Couldn't resolve original symbol for ./lib/core/directives/ng-vFor-container.directive from /Users/XXXXXXX/Desktop/starter-kit/node_modules/ngvforlib/package/ng-vfor-lib.d.ts

I corrected the naming in the doc on github, i was making some changes before publishing... See below

I used the new Angular 6 cli with packgr, currently it means that it needs angular 6 to work. I was thinking of redoing with a @<6.0 and also the @~6.1 versions of angular.

currently the peers it requires are:

  "peerDependencies": {
    "@angular/common": "^6.1.0",
    "@angular/core": "^6.1.0"
  },

Which version of angular are you guys using ?

Also,

the npm is ngvforlib, the include is:

import { NgGUDCoreModule } from 'ng-vfor-lib';

**Comment by OrangeDrangon** [comment]
Thanks... 6 is fine with me

**Comment by pablomaurer** [comment]
angular 5 because of ionic 3.
ionic 4 with angular 5 is in beta and until they reach some usable state goes probably until ionic 4.2

**Comment by Marcidius** [comment]
The problem is that when you do an install with `npm i ngvforlib` the files that are actually installed are the following:

1. ng-vfor-lib-0.0.5.tgz
2. package.json
3. README.md

It appears there is a problem with the package from NPM.  I was hoping to use this module and give it a go in our large Angular project but no dice.

**Comment by OrangeDrangon** [comment]
Yeah they published a zipped package. It may be possible to extract it and use it.

EDIT: After installing the package you can navigate into the node_modules folder and unzip the package yourself and it works

**Comment by Shira16** [comment]
I have the same problem. I unpacked zipped package but still getting: 
ERROR in ./src/app/app.module.ngfactory.js
Module not found: Error: Can't resolve 'ng-vfor-lib'

I am using angular 6.1.6

**Comment by Hadard753** [comment]
+1 still getting this error 
I used : 
npm install ngvforlib
import { NgGUDCoreModule } from 'ng-vfor-lib';

and using angular 7 
Im getting ERROR : "cannot find module ng-vfor-lib"
and I dont have ng-vfor-lib directory under node_modules only ngvforlib directory which only contains three files:   ng-vfor-lib-0.0.9.tgz, readme.md and package.json

**Comment by kyledenton** [comment]
> +1 still getting this error
> I used :
> npm install ngvforlib
> import { NgGUDCoreModule } from 'ng-vfor-lib';
> 
> and using angular 7
> Im getting ERROR : "cannot find module ng-vfor-lib"
> and I dont have ng-vfor-lib directory under node_modules only ngvforlib directory which only contains three files: ng-vfor-lib-0.0.9.tgz, readme.md and package.json

Same happens for me.  This package doesn't install correctly which makes the application useless.  It will fail on a build server and it will fail for any other developer who doesn't know they have to go in and unzip the files ( which doesn't work anyway ).

**Comment by daniel-halldorsson** [comment]
+1 When trying to install on Angular 8. Same, was really looking forward to trying this library, sounds perfect and much needed as none of the other virtual scrolling solutions are versatile enough, im guessing development has stopped on it by the looks of things?

**Comment by tacho090** [comment]
+1 Same issue here with:

Angular CLI: 8.2.1
Node: 10.16.0
OS: linux x64
Angular: 5.2.11

What a shame, the package looked spectacular! I suppose development stopped since there's no response from the devs.
